src/main/java/com/lewisd/maven/lint/rules/AbstractReduntantVersionRule.java

```diff
@@ -47,20 +47,20 @@ protected void checkForRedundantVersions(final MavenProject mavenProject,
         }
 
         String version = resolveVersion(modelObject, resolvedModelObject);
-        String inheritedVersion = modelUtil.getVersion(tryResolveObject(inheritedObject));
+        String inheritedVersion = getModelUtil().getVersion(tryResolveObject(inheritedObject));

```

Comment:
Lol. If you go leaving getters/setters you leave object-oriented pattern. I think litzkov-law matters. Changing fundamental styles is difficult.